changelog.json

```
@@ -1,5 +1,10 @@
 {
   "releases": {
+    "3.0.0": [
+      "[New] Add support for notifications of pull request reviews - #14175",
+      "[Fixed] Use fast-forward flag as default for pulling divergent paths - #14431",
```

This is fine for beta but a little bit to focused on the technical aspect for prod

```suggestion
      "[Fixed] Default to merging when pulling without a configured preference between merge or rebase - #14431",
```

---

SGTM